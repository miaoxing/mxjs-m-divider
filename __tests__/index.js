import Divider from '..';
import {render} from '@testing-library/react';

describe('Divider', () => {
  test('basic', () => {
    const {container} = render(<Divider/>);
    expect(container).toMatchSnapshot();
  });
});
