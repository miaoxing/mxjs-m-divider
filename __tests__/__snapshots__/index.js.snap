// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider basic 1`] = `
<div>
  <taro-view-core
    class="mx-divider"
  />
</div>
`;
